src/SOS/SOS.UnitTests/ConfigFiles/Unix/Debugger.Tests.Config.txt

```diff
@@ -106,6 +108,7 @@
       <BuildProjectFramework>$(TargetFrameworkLatest)</BuildProjectFramework>
       <DotNetDiagnosticExtensions>$(RootBinDir)/bin/TestExtension/$(TargetConfiguration)/netstandard2.0/TestExtension.dll</DotNetDiagnosticExtensions>
       <SetHostRuntime>$(DotNetRoot)/shared/Microsoft.NETCore.App/$(RuntimeFrameworkVersion)</SetHostRuntime>
+      <BuildType>Nightly</BuildType>
```

Comment:
I wish there was a better way to do this than putting in the config file. RuntimeVersionLast is setting to the latest/nightly version.  You could put code in SOSRunner.cs to get that config value and compare it to the RuntimeFrameworkVersion config property